app/models/notification.rb

```diff
@@ -62,8 +62,8 @@ def set_default_initiator
   end
 
   def set_notification_recipients
-    subscribers = subscriber_ids
-    notification_recipients.build subscribers.map{|id| { :user_id => id}}
+    subscribers = User.unscoped.where(:id => subscriber_ids)
+    notification_recipients.build subscribers.map{|user| { :user => user}}
```

Comment:
for reviewer (and myself): If we used `:user_id`, rails later tried to load the association when it validates presence of `user` but it respects the default scope defined by taxonomix, therefore it couldn't find the user and silently failed to save the built associated notification recipient. Searching for users like this should be more efficient, since it loads all of them in one query.